app/src/myanmar/AndroidManifest.xml

```diff
@@ -13,7 +13,7 @@
 
         <provider
             android:name=".data.database.utils.SCMMDBProvider"
-            android:authorities="${applicationId}.data.database.utils.SCMMDBProvider"
+            android:authorities="org.eyeseetea.malariacare.data.database.utils.SCMMDBProvider"
```

Comment:
applicationId is necessary to avoid provider conflict with buddybuild app installation, Why you need remove it?